### PR TITLE
chore: i18n support for service registry art Artifact commands

### LINKED ISSUE #904: I18n support for service registry art Artifact commands
### Feature or problem description

Add i18n support for artefact commands

### PR DESCRIPTION
**Pull request by alexal**
Closes #904

### CODE REVIEW

**Comment by wtrocki** [comment]
WOW :)

File: pkg/cmd/registry/artifact/crud/create/create.go [hunk 1]
```diff
@@ -83,17 +56,11 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
...
-		Long:  longDescription,
-		Example: `
-# Create an artifact in default group
-rhoas service-registry artifact create my-artifact.json
-
-# Create an artifact with specified type
-rhoas service-registry artifact create --type=JSON my-artifact.json
-		`,
-		Args: cobra.RangeArgs(0, 1),
+		Use:     f.Localizer.MustLocalize("artifact.cmd.create.use"),
```
Comment (wtrocki) [inline pkg/cmd/registry/artifact/crud/create/create.go:59]: I did not internationalise use but I think we aren'tp consistent in this across cli. 
Command names should not change

**Review by wtrocki — APPROVED** [review]
All seems good. Need to test every cmd tommorow

**Comment by alexal** [comment]
@wtrocki you're welcome. Let me know if something needs to be changed.

File: pkg/cmd/registry/artifact/crud/create/create.go (hunk 1 above)
Comment (alexal) [reply pkg/cmd/registry/artifact/crud/create/create.go:59]: Let me know if you want to revert strings for "use".

Comment (wtrocki) [reply pkg/cmd/registry/artifact/crud/create/create.go:59]: No need. All good.

**Pull request merged**